### DISCUSSION TITLE
Subsetting for models other than the HRRR

### DISCUSSION THREAD

**Discussion by BrandonWeart** [blaylockbk/Herbie#331] [Q&A]
Is subsetting supported for any of the other models or is it just the HRRR?

**Comment by blaylockbk** [comment]
Can you clarify what you mean by "subsetting"? If you mean _subsetting by variables_? Then yes, as long as the GRIB file has an index associated with it.

The tutorial for GFS shows subsetting the 2-m temperature data from GFS
https://herbie.readthedocs.io/en/stable/user_guide/tutorial/model_notebooks/gfs.html
